pkg/controllers/provisioningv2/rke2/machineprovision/controller.go

```diff
@@ -413,18 +422,65 @@ func (h *handler) doRemove(infraObj *infraObject) (runtime.Object, error) {
 	return obj, generic.ErrSkip
 }
 
+func (h *handler) EnqueueAfter(infra *infraObject, duration time.Duration) {
+	err := h.dynamic.EnqueueAfter(infra.obj.GetObjectKind().GroupVersionKind(), infra.meta.GetNamespace(), infra.meta.GetName(), duration)
+	if err != nil {
```

Comment:
Should we do something with this error other than just logging it?

Reply:
I originally had a panic, since this is the summation of `EnqueueAfter`:
https://github.com/rancher/lasso/blob/6ea88ca2fbcc8cadc89c30421b0766e9993c4b7a/pkg/dynamic/controller.go#L349-L357
If this happens, everything is spectacularly broken.